README.md

```diff
@@ -1,19 +1,19 @@
 # Benchmark.js v2.1.0
 
-A [robust](http://calendar.perfplanet.com/2010/bulletproof-javascript-benchmarks/ "Bulletproof JavaScript benchmarks") benchmarking library that supports high-resolution timers & returns statistically significant results. As seen on [jsPerf](http://jsperf.com/).
+A [robust](https://mathiasbynens.be/notes/javascript-benchmarking "Bulletproof JavaScript benchmarks") benchmarking library that supports high-resolution timers & returns statistically significant results. As seen on [jsPerf](https://jsperf.com/).
 
 ## Documentation
 
-* [API Documentation](http://benchmarkjs.com/docs)
+* [API Documentation](https://benchmarkjs.com/docs)
 
 ## Download
 
- * [Development source](https://raw.github.com/bestiejs/benchmark.js/v2.1.0/benchmark.js)
+ * [Development source](https://raw.githubusercontent.com/bestiejs/benchmark.js/2.1.0/benchmark.js)
```

Comment:
This URL 404’d. The tag didn’t exist yet, so I’ve created it. Also we seem to use the `2.1.0` format (no `v` prefix).